src/app/shared/components/alert/alert.component.html

```diff
@@ -1,14 +1,22 @@
 <div class="alert alert--dark">
   <div class="alert__icon">
-    <svg-icon src="assets/images/icon-sprite.svg#error" class="icon"></svg-icon>
+    <ng-template [ngIf]="icon == 'error'">
+      <svg-icon
+        src="assets/images/icon-sprite.svg#error"
+        class="icon"
+      ></svg-icon>
+    </ng-template>
+    <ng-template [ngIf]="icon == 'notification'">
+      <svg-icon
+        src="assets/images/icon-sprite.svg#notification"
+        class="icon"
+      ></svg-icon>
+    </ng-template>
```

Comment:
@junan How about like this?

```suggestion
    <ng-template>
      <svg-icon
        src="assets/images/icon-sprite.svg#{{icon}}"
        class="icon"
      ></svg-icon>
    </ng-template>
```

Reply:
I tried it, it does not works. Seems `svg-icon-sprite` does not work with the dynamic path

Reply:
@junan I found the way to make it dynamic 😎 , their [document](https://github.com/jannicz/svg-icon-sprite/blob/master/INTEGRATION.md#angular) is not quite clear but there's a [code sandbox](https://codesandbox.io/s/happy-chatelet-zsl93) which provide the example that we can use `[attr.src]` to set dynamic source. 🙌🏼  

```suggestion
    <ng-template>
      <svg-icon
        [attr.src]="'assets/images/icon-sprite.svg#'+alertType"
        class="icon"
      ></svg-icon>
    </ng-template>
```

Reply:
@carryall Great, thanks!